planner/core/task.go

```diff
@@ -1977,6 +1977,10 @@ func (p *PhysicalHashAgg) attach2TaskForMpp(tasks ...task) task {
 		}
 		attachPlan2Task(proj, newMpp)
 		return newMpp
+	case NoMpp:
```

Comment:
Why make this change?

Reply:
To avoid goup by query give "cannot find proper physical plan" bug.